test/node.js

```diff
@@ -341,6 +363,13 @@ node.randomTxAccount = function () {
 	});
 };
 
+// Returns an random basic transaction to send 1 LSK from genesis account to a random account
+node.randomTx = function (offset) {
+	var randomAccount = node.randomAccount();
+
```

Comment:
Remove empty line

Reply:
We always leave one empty line between the definition of variables and execution/return.